mp4ra_FFV1.md

```diff
@@ -0,0 +1,34 @@
+# Mapping FFV1 into the MP4
+
+This document represents preparation notes for a request to mp4ra.org to register FFV1 for use in MP4, following the instructions at the [mp4ra.org request page](http://www.mp4ra.org/request.html).
+
+## Email Request
+
+Draft of email.
+
+### For registration with ISO
+to: mp4reg@group.apple.com
+subject: mp4reg FFV1
+
+### For registration with QuickTime
+to: qtfourcc@group.apple.com for registration with QuickTime
+subject: qtreg FFV1
+
+cc: ffmpeg-devel?
+
+### Email Message
+
+[Name] of [address] at [organization url] requests the registration of an identifier for a lossless video encoding, FFV1, which is presently under standardization within the IETF [cellar working group](https://datatracker.ietf.org/wg/cellar/charter/).
+
+This request is for a codec kind of codepoint with a video stream type. The recommended identifier is "FFV1". The specification was initially published by FFmpeg at http://ffmpeg.org/~michael/ffv1.html. Presently the specification is under development by the IETF cellar working group and the most up-to-date copies of the specification are available at https://datatracker.ietf.org/doc/draft-niedermayer-cellar-ffv1/ (possibly changing to https://datatracker.ietf.org/doc/draft-ietf-cellar-ffv1/ later). Development work on the specification may be viewed at https://github.com/FFmpeg/FFV1.
+
```

Comment:
Shouldn't we put somewhere that the init data (ConfigurationRecord) is in the "glbl" atom (beside "fiel", "esds", colr... atoms)?